articles/azure-monitor/app/java-get-started.md

```diff
@@ -164,6 +164,8 @@ You can also [set it in code](../../azure-monitor/app/api-custom-events-metrics.
     }
 ```
 
+Please note that [Live Metrics](https://docs.microsoft.com/azure/azure-monitor/app/live-stream) does not support reading instrumentation key from code.
```

Comment:
https://github.com/microsoft/ApplicationInsights-Java/issues/969 link to actual. Dont want it part of docs, but pasting here for reviewer's reference.